string device_model
string robot_conf

float64 min_range
float64 max_range

float64 knee_range
float64 var
float64 perc

float64[] dead_regions
float64[] upwards_mirror_regions
float64[] downwards_mirror_regions

float64[12] relative_pose
